LayoutTests/imported/w3c/resources/import-expectations.json

```diff
@@ -103,6 +103,8 @@
     "web-platform-tests/css/css-shapes": "import",
     "web-platform-tests/css/css-shapes/test-plan/index.html": "skip",
     "web-platform-tests/css/css-sizing/aspect-ratio": "import",
+    "web-platform-tests/css/css-sizing/block-size-with-min-or-max-content-table-1a.html": "import",
+    "web-platform-tests/css/css-sizing/block-size-with-min-or-max-content-table-1b.html": "import",
```

Comment:
I'd much prefer if the whole `web-platform-tests/css/css-sizing/` directory was imported. Not just these 2 tests.